apps/nextra/pages/en/build/cli/install-cli/install-cli-mac.mdx

```diff
@@ -0,0 +1,36 @@
+import { Callout, Steps } from 'nextra/components'
+
+# Install the Aptos CLI on Mac
+
+For Mac, the easiest way to install the Aptos CLI is with the package manager `brew`.
+
+# Installation
+
+<Steps>
+### Ensure you have `brew` installed https://brew.sh/.
+### Open a new terminal and enter the following commands.
+
+```bash filename="Terminal"
+brew update
+brew install aptos
+```
+
+### Open another terminal and run `aptos help` to verify the CLI is installed.
+
+```bash filename="Terminal"
+aptos help
+```
+</Steps>
+
+<Callout type="warning" emoji="⚠️">
```

Comment:
Let's go forward with this, and we'll replace all of these with shared components

Reply:
I think it'll render correctly without the `emoji` property

Reply:
It does indeed :)